docs/en/operations/server-configuration-parameters/_server_settings_outside_source.md

```diff
@@ -0,0 +1,2488 @@
+## asynchronous_metric_log {#asynchronous_metric_log}
```

Comment:
This file now contains all the server settings which were previously in `settings.md` ([server settings](https://clickhouse.com/docs/operations/server-configuration-parameters/settings) in the docs currently) minus the ones which are in `system.server_settings` i.e it only contains the server settings not documented in the source.